src/libPMacc/include/pluginSystem/PluginConnector.hpp

```diff
@@ -180,6 +180,19 @@ namespace PMacc
             }
         }
 
+        template<typename Plugin>
+        Plugin* getPluginFromType()
```

Comment:
Yes please move this to an own pull feature request and add a description.

This method only return the first plugin of the type. Is it possible to have more than one plugin with the given type in list  named `plugins`?